docs/interface.md

```diff
@@ -30,9 +30,9 @@ partial interface WorkerNavigator {
 };
 
 dictionary ShareData {
-  DOMString? title;
-  DOMString? text;
-  DOMString? url;
+  DOMString title;
+  DOMString text;
+  USVString url;
```

Comment:
This is rather unique and without precedent I think, but it sounds like a good reason to avoid surrogates.